editor-services.json

```diff
@@ -1,3 +1,5 @@
 {
-  "release": "0.14.0"
+  "githubuser": "lingua-pupuli",
+  "githubrepo": "puppet-editor-services",
+  "githubref": "master"
```

Comment:
The PR says update to 0.15.0, but this says master. Are we moving to gitrefs or is this just until 0.15 is 'released'?

Reply:
Just until 0.15 is released. Which is why this is a WIP PR still.